lxd/storage_pools.go

```diff
@@ -136,7 +136,8 @@ func storagePoolsPost(d *Daemon, r *http.Request) Response {
 	// storage config are the ones in StoragePoolNodeConfigKeys.
 	for key := range req.Config {
 		if !shared.StringInSlice(key, db.StoragePoolNodeConfigKeys) {
-			return SmartError(fmt.Errorf("Invalid config key '%s'", key))
+			return SmartError(
+				fmt.Errorf("Config key '%s' may not be used as node-specific key", key))
```

Comment:
Can we avoid the wrapping here, makes grepping unnecessarily harder in this case (we don't have a max-width requirement in our codebase)?

Reply:
Done, hope it's what you meant.